dev/TeachingTip/TestUI/TeachingTipPage.xaml

```diff
@@ -43,29 +43,6 @@
                         </muxc:TeachingTip.Content>
                     </muxc:TeachingTip>
                     <Button x:Name="targetButton" HorizontalAlignment="Center" VerticalAlignment="Center" Click="OnShowButtonClicked" Height="300">Enable AutoSave
-                        <muxc:TeachingTip.Attach>
-                            <muxc:TeachingTip x:Name="TeachingTipInSetAttach" 
```

Comment:
>TeachingTipInSetAttach [](start = 54, length = 22)

Can this sample be changed to a Target="{x:Bind ...}" setter in a ResourceDictionary ? Then it still verifies what you want of being able to "attach" the teaching tip in markup.